agnpy/utils/sedintegrable.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+import astropy.units as u
+
+
+class SedFluxIntegrable:
+    def integrate_flux(self, nu_min, nu_max, nu_points=50):
+        r""" Evaluates the SED flux integral over the span of provided frequencies
+
+        Parameters
+        ----------
+        nu_min : start frequency (in Hz or equivalent)
+        nu_max : end frequency (in Hz or equivalent)
+        nu_points: number of points (between nu_min and nu_max) on the log scale
+        """
+        nu = self._nu_logspace(nu_min, nu_max, nu_points)
+        sed_nufnu = self.sed_flux(nu)  # erg / s / cm2 / log(nu)
+        sed_fnu = sed_nufnu / nu       # erg / s / cm2 / Hz
+        return np.trapz(sed_fnu, nu)   # erg / s / cm2
+
+    def integrate_photon_flux(self, nu_min, nu_max, nu_points=50):
+        r""" Evaluates the photon flux integral from SED over the span of provided frequencies
+
+        Parameters
+        ----------
+        nu_min : start frequency (in Hz or equivalent)
+        nu_max : end frequency (in Hz or equivalent)
+        nu_points: number of points (between nu_min and nu_max) on the log scale
+        """
+        nu = self._nu_logspace(nu_min, nu_max, nu_points)
```

Comment:
The variable is actually called just "nu" :)

The helper function I called _nu_logspace(), because it's a small wrapper around np.logspace(), would you prefer me to rename it to "_nu()" or something similar ? (unless we remove the function completely)